packages/apps-config/src/api/spec/subspace.ts

```diff
@@ -119,13 +112,9 @@ const definitions: OverrideBundleDefinition = {
     {
       minmax: [0, undefined],
       types: {
-        FarmerPublicKey: '[u8; 32]',
+        FarmerPublicKey: 'AccountId32',
```

Comment:
Are these definitions even necessary with Metadata v14?

Reply:
Not in general, but in this case, I have it defined so the API can handle this field as an account type with the createType. 
This way I can set the author as an account type directly.

Reply:
I think that I can just remove the FarmerPublicKey definitions and treat the solution.public_key field directly as an accountId. Will make the code simpler. Something like this.
```
types: {
        Solution: {
          public_key: 'AccountId32'
        },
        SubPreDigest: {
          slot: 'u64',
          solution: 'Solution'
        }
      }
```

Reply:
> I have it defined so the API can handle this field as an account type with the createType

I still don't get why it is needed if the same exact information will be downloaded from node's metadata automatically.

Reply:
For the solution at least I just get it as Bytes using the metadata. So I need to declare it in here otherwise the create type fails decoding it.

Reply:
I checked the complete lookup section yesterday and seems like there are still some missing types on v14, especially for structs used inside another. Like this case